charts/zora/values.yaml

```diff
@@ -197,6 +203,9 @@ scan:
       ignoreDescriptions: false
       # -- [Resources](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers) to add to `trivy` container
       resources: {}
+      # -- Annotations added to the trivy pods
+      podAnnotations:
+        trivy-annotation: trivy-value
```

Comment:
```suggestion
        #trivy-annotation: trivy-value
```

Reply:
I already tried this, but it caused the README to be generated as a string.  Leaving as is for now.